dspace/config/emails/dissertation_notify_archive

```diff
@@ -0,0 +1,126 @@
+    </p>
+    <h1>DIAL.pr</h1>
+    <p class="timestamp">${params[0]}</p>
+    <div class="language" style="border-bottom:1px solid #777; color: #777;">
+      Version française (English version below)
+    </div>
+    <p>Madame/Monsieur,</p>
+    <p>
+      Suite à votre réussite, votre thèse a bien été publiée dans DIAL.pr.<br/>
+    </p>
+    <h4 class="heading">Informations relatives à la thèse</h4>
+    <table class="metadata-table" style="border: 1px solid #777">
+      <tbody>
+        #foreach ($mapEntry in $params[2].entrySet())
+          <tr>
+            <th style="border-right: 1px solid #777">$mapEntry.key</th>
+            <td>$mapEntry.value</td>
+          </tr>
+        #end
+        <tr>
+          <th style="border-right: 1px solid #777">Lien vers la thèse</th>
+          <td>
+            <a href="${params[1]}">${params[1]}</a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <h4 class="heading">Fichiers attachés</h4>
+    <ul>
+      #foreach ($attached_file in $params[4])
+        <li>$attached_file[0] ($attached_file[1])</li>
+      #end
+    </ul>
+    #if ($params[4].isEmpty())
+      <i>Aucun fichier attaché trouvé</i>
+    #end
```

Comment:
Simplify --> if there are some file, display them ; otherwise don't display the section